api/hooks/bannedItems/index.js

```diff
@@ -31,8 +31,8 @@ module.exports = function banneditems(sails) {
     stop: stop
   };
 
-  function start(serverId) {
-    const loggingObject = sails.hooks.sdtdlogs.getLoggingObject(serverId);
+  async function start(serverId) {
```

Comment:
stop wasn't updated to await
also stop is going to getLoggingObject, start it if its not existing, then stop it
so maybe you want to add getLoggingObject(serverId, { startIfNotExists: false }) or something for stop?

Reply:
This is a minor optimization, I don't think it'll make much difference and it'd make `getLoggingObject()` even more complex than it already is. Not worth IMO unless you strongly feel it'd make a difference

Reply:
Well stop right now isn't async and calling an async function so will crash